api/src/main/java/uk/gov/hmcts/payment/api/controllers/CreditAccountPaymentController.java

```diff
@@ -134,15 +122,15 @@ public CreditAccountPaymentController(@Qualifier("loggingCreditAccountPaymentSer
         }
 
         // trigger Apportion based on the launch darkly feature flag
-        boolean apportionFeature = featureToggler.getBooleanValue("apportion-feature",false);
+        boolean apportionFeature = featureToggler.getBooleanValue("apportion-feature", false);
         LOG.info("ApportionFeature Flag Value in CreditAccountPaymentController : {}", apportionFeature);
-        if(apportionFeature) {
+        if (apportionFeature) {
             Payment pbaPayment = paymentFeeLink.getPayments().get(0);
             pbaPayment.setPaymentLink(paymentFeeLink);
             feePayApportionService.processApportion(pbaPayment);
 
             // Update Fee Amount Due as Payment Status received from PBA Payment as SUCCESS
-            if(Lists.newArrayList("success", "pending").contains(pbaPayment.getPaymentStatus().getName().toLowerCase())) {
+            if (Lists.newArrayList("success", "pending").contains(pbaPayment.getPaymentStatus().getName().toLowerCase())) {
```

Comment:
Please remove the pending from the condition.

Reply:
After removing it, it caused drop in coverage for odd reason. Will keep this for now and remove it at later stage